### ISSUE TITLE
Doesn't work anymore in VSCode 1.63.1

### ISSUE THREAD

**Issue by gremo** [wongjn/vscode-php-sniffer#60]
I love this extension but it seems not working anymore in the recent VSCode version. I can't find any reference in logs/output window, it just doesn't show errors and auto-formatting isn't working anymore.

Workspace settings:

```json
{
  "[php]": {
    "editor.defaultFormatter": "wongjn.php-sniffer",
    "editor.formatOnSave": true
  },
  "phpSniffer.autoDetect": true,
  "phpSniffer.run": "onType",
}
```

My `phpcs.xml.dist`:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
    <arg name="colors" />
    <arg name="extensions" value="php" />

    <file>deploy.php</file>
    <file>scheduler.php</file>
    <file>src/</file>
    <file>tests/</file>
    <exclude-pattern>config/bootstrap.php</exclude-pattern>
    <exclude-pattern>config/preload.php</exclude-pattern>
    <exclude-pattern>tests/bootstrap.php</exclude-pattern>
    <exclude-pattern>var/</exclude-pattern>
    <exclude-pattern>vendor/</exclude-pattern>

    <rule ref="PSR12" />
</ruleset>
```

**Comment by wongjn** [comment]
Anything in devtools console perhaps? Help → Toggle Developer Tools → Console

**Comment by gremo** [comment]
Well actually yes, but is something related to phpstan extension. But even if I disable phpstan, your extension still doesn't work anymore (no warnings and no auto formatting):

![console](https://user-images.githubusercontent.com/1532616/146364719-fae26ebc-9feb-44d6-a8b9-68fe1f1613b0.png)

**Comment by wongjn** [comment]
Could you share a minimal reproduction repository please?

**Comment by gremo** [comment]
Sure, here we are: https://github.com/gremo/vscode-php-sniffer-sample

As you can see from the project, it seems _not working for some particular rules_.

If you run `vendor/bin/phpcs` you see the violation of `PSR12.Operators.OperatorSpacing.NoSpaceBefore` and `PSR12.Operators.OperatorSpacing.NoSpaceAfter`. The extension doesn't show any error and the auto-fix formatting doesn't work either (you can still fix the problem running `vendor/bin/phpcbf`). 
![phpcs](https://user-images.githubusercontent.com/1532616/146368930-48bc11dc-a0f3-4a65-be47-bb026b135a5a.png)

Hope this help and thank you for your work!
